### ISSUE TITLE
Update `rosbridge_suite` and `roslibjs`

### ISSUE THREAD

**Issue by amalnanavati** [personalrobotics/feeding_web_interface#97]
Upstream changes to make `rosbridge_suite` compatible with ROS2 actions have [already been merged in](https://github.com/RobotWebTools/rosbridge_suite/pull/813#event-10927112244), and there is [an open PR](https://github.com/RobotWebTools/roslibjs/pull/645) to do the same for `roslibjs`. Once that is merged in, we should update our system to use the upstream code. (Note: we have to use this together, because there were some additional changes in the upstream PR for `rosbridge_suite` e.g., renaming `send_goal` to `send_action_goal`.)

**Comment by amalnanavati** [comment]
Update: although ROS2 actions support has been merged into [rosbridge](https://github.com/RobotWebTools/rosbridge_suite/pull/886) and [roslibjs](https://github.com/RobotWebTools/roslibjs/pull/645), there is still a [lingering issue](https://github.com/RobotWebTools/rosbridge_suite/issues/909) where asynchronous action cancellation is not possible if that action was also initiated by the web app. Thus, we cannot yet switch to the upstream code.
